### PR TITLE
proxy auth headers complementary documentation

### PR DESCRIPTION
**Pull request by renato-farias**
This PR adds complementary documentation on how to setup auth headers passed by proxies.

Signed-off-by: Renato Farias <renatocdfs@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -435,6 +435,22 @@ passing it as the `BASE_URL` environment variable.
 
 When using an authentication proxy, Terraboard will retrieve the logged in
 user and email from the headers passed by the proxy.
+Terraboard expects you to setup the HTTP Headers `X-Forwarded-User` and
+`X-Forwarded-Email` when passing the llogged in user and email. A Nginx
```
Comment (bmpandrade) [inline README.md:439]: A typo on `llogged` -> `logged`.

Comment (renato-farias) [reply README.md]: Fixed. Thanks for reviewing it.

File: README.md [hunk 2]
```diff
@@ -435,6 +435,22 @@ passing it as the `BASE_URL` environment variable.
...
+
+```nginx
+location / {
+  ....
+  auth_request_set $user   $upstream_http_x_auth_request_user;
+  auth_request_set $email  $upstream_http_x_auth_request_email;
+  proxy_set_header X-Forwarded-User  $user;
+  proxy_set_header X-Forwarded-Email $email;
+  ...
+  proxy_pass http://terrabord/;
```
[the remark this answers is not in the document]
Comment (renato-farias) [reply README.md]: Fixed. Thanks for reviewing it.

**Comment by hbollon** [comment]
Thanks for your contribution!

**Pull request merged**